src/include/dhcp-server/widgets.rb

```diff
@@ -325,7 +326,7 @@ def OpenFirewallValidate(id, event)
         Builtins.foreach(@ifaces) do |ifcfg, interface|
           # interface is active
           if Ops.get_boolean(interface, "active", false) == true
-            if SuSEFirewall.GetZoneOfInterface(ifcfg) == nil
+            unless firewalld.zones.find { |z| z.interfaces.include?(ifcfg) }
```

Comment:
np: I prefer any? 

Although would prefer to use `unless interface_zone(ifcfg)`